apps/core/components/userContext/Firebase.js

```diff
@@ -0,0 +1,25 @@
+// @flow
+
+import firebase from 'firebase/app';
+import 'firebase/auth';
+import {
+  FIREBASE_API_KEY,
+  FIREBASE_AUTH_DOMAIN,
+  FIREBASE_DATABASE_URL,
+  FIREBASE_PROJECT_ID,
+  FIREBASE_STORAGE_BUCKET,
+  FIREBASE_MESSAGING_SENDER_ID,
+} from 'react-native-dotenv';
+
+const CONFIG = {
+  apiKey: FIREBASE_API_KEY,
+  authDomain: FIREBASE_AUTH_DOMAIN,
+  databaseURL: FIREBASE_DATABASE_URL,
+  projectId: FIREBASE_PROJECT_ID,
+  storageBucket: FIREBASE_STORAGE_BUCKET,
+  messagingSenderId: FIREBASE_MESSAGING_SENDER_ID,
+};
+
+if (!firebase.apps.length && FIREBASE_API_KEY.length > 0) {
```

Comment:
Not sure about this condition being necessary.

Reply:
True, it's not really clear at first glance. I've added short description to this condition to make clear why it is necessary.